internal/config/helpers.go

```diff
@@ -34,6 +34,15 @@ func InProdClowder() bool {
 	return clowder.IsClowderEnabled() && strings.HasPrefix(*clowder.LoadedConfig.Metadata.EnvName, "env-stage")
 }
 
+// RequestedTopic returns mapped topic from Clowder. When not running in Clowder mode, it returns the input topic name.
+func RequestedTopic(topic string) string {
```

Comment:
`KafkaTopic` or `RealKafkaTopicName` might be better names here, but no biggie :)

Reply:
Yeah I try to stick with the clowder terminology here, not that I like it but should be less confusing. The function does map requested topics to (real) topics, which I would just call "topics" I guess. Since in Go we should avoid getters, "GetTopic" would be bad, so it would end up as "Topic" or "RealTopic" which sounds non-descriptive to me.